src/pages/wallet/Buy.tsx

```diff
@@ -54,6 +55,11 @@ export const exchanges = {
       href: "https://trading.bitfinex.com/t/TERRAUST:USD",
       icon: <Bitfinex width={24} height={24} />,
     },
+    {
+      children: "Kraken",
+      href: "https://www.kraken.com/u/trade?receive=UST&spend=USD",
```

Comment:
https://trade.kraken.com/charts/KRAKEN:UST-USD
Why don't you use this domain?
Because the domain you are providing requires sign in first.